docs/_getting-started/index.md

```diff
@@ -449,4 +449,4 @@ following this with the [Geth with Clef](/docs/getting-started/geth_with_clef) t
 adopt more secure account management practices than those outlined here.
 
 
-[cli]: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line
```

Comment:
The URL is same. Can you please revert this line of change?

Reply:
IIUC it just adds a linebreak